django/conf/global_settings.py

```diff
@@ -492,6 +492,7 @@
 PASSWORD_HASHERS = [
     'django.contrib.auth.hashers.PBKDF2PasswordHasher',
     'django.contrib.auth.hashers.PBKDF2SHA1PasswordHasher',
+    'django.contrib.auth.hashers.Argon2PasswordHasher',
```

Comment:
I think this ordering is reasonable. PBKDF2 should be the default preferred hasher, because it is always usable without requiring a third-party package. All of the bundled hashers should be in the default list somewhere, even though it has no real immediate effect, both because it provides additional "documentation" of their existence, and for the reason @dstufft noted (if someone tries out the argon2 hasher in a custom hashers setting and then deletes their custom setting, those passwords should still be upgradable via the default setting).